### PR TITLE
Seperate dashboard usage from development

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: dashboard-dev/install.bash [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env bash
+
+github_url="https://github.com/tue-robotics/dashboard.git"
+dest="$HOME/src/dashboard"
+
+tue-install-git "$github_url" "$dest"
```
Comment (ar13pit) [inline dashboard-dev/install.bash:6]: Dashboard is a catkin package. Either change the install type or remove package.xml and CMakeLists.txt from the dashboard repository.

Comment (MatthijsBurgh) [reply dashboard-dev/install.bash:6]: Agreed, I will remove it.

File: tue-dev/install.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@
 # -------------------- ROBOTS ---------------------
 
 - type: target
-  name: ros-dashboard
+  name: dashboard
```
Comment (ar13pit) [inline tue-dev/install.yaml:33]: What about dashboard-dev?

Comment (MatthijsBurgh) [reply tue-dev/install.yaml:33]: A person needs to install it manually, when he/she wants to develop it.

**Pull request merged**